.env.sample

```diff
@@ -195,10 +195,10 @@ BING_API_KEY=
 GOOGLE_API_KEY=
 
 # Monitoring
-MONITORING_ENABLED=True
+MONITORING_ENABLED=False
 MONITORING_DATA_TTL=365
-USER_ANALYTICS_ENABLED=True
-USER_ANALYTICS_GZIP=True
+USER_ANALYTICS_ENABLED=False
```

Comment:
@afabiani I would leave this `True`. It won't have any effect if monitoring is disabled.

Reply:
@giohappy done